src/error/RecordNotFoundError.ts

```diff
@@ -0,0 +1,7 @@
+import { AriesFrameworkError } from './AriesFrameworkError'
```

Comment:
Off-topic: I like the import `from './AriesFrameworkError'` and not `from '.'` as it's at some other places. It would be good to have some convention and keep with that. What do you think?

The convention would be: If you import from the same module you should import by the whole path. If you import from a different module you can import from `index.ts`

Reply:
I agree with this convention. I'll try to maintain this convention going forward